### PR TITLE
Bug: WER sidecar info not appearing in SBS

### PR DESCRIPTION
**Pull request by nishchalb**

### CODE REVIEW

File: src/Nlp.cpp [hunk 1]
```diff
@@ -28,26 +28,25 @@ NlpFstLoader::NlpFstLoader(std::vector<RawNlpRecord> &records, Json::Value norma
...
     auto curr_row_tags = row.wer_tags;
 
     // Update wer tags in records to real string labels
-    vector<string> real_wer_tags;
     for (auto &tag : curr_row_tags) {
-      auto real_tag = tag;
       if (mWerSidecar != Json::nullValue) {
-        real_tag = "###" + real_tag + "_" + mWerSidecar[real_tag]["entity_type"].asString() + "###";
+        tag.entity_type = mWerSidecar[tag.tag_id]["entity_type"].asString();
+        logger->info(tag.entity_type);
```
Comment (qmac) [inline src/Nlp.cpp:44]: too verbose

File: src/Nlp.cpp [hunk 2]
```diff
@@ -28,26 +28,25 @@ NlpFstLoader::NlpFstLoader(std::vector<RawNlpRecord> &records, Json::Value norma
...
     auto punctuation = row.punctuation;
     auto curr_row_tags = row.wer_tags;
 
     // Update wer tags in records to real string labels
-    vector<string> real_wer_tags;
     for (auto &tag : curr_row_tags) {
-      auto real_tag = tag;
       if (mWerSidecar != Json::nullValue) {
-        real_tag = "###" + real_tag + "_" + mWerSidecar[real_tag]["entity_type"].asString() + "###";
+        tag.entity_type = mWerSidecar[tag.tag_id]["entity_type"].asString();
```
Comment (qmac) [inline src/Nlp.cpp:43]: what happened to the `###` stuff

File: src/Nlp.cpp (hunk 1 above)
Comment (nishchalb) [reply src/Nlp.cpp:44]: Ah sorry, that was extra debug. should be removed

File: src/Nlp.cpp (hunk 2 above)
Comment (nishchalb) [reply src/Nlp.cpp:43]: It's been delayed to the `WriteSbs` function https://github.com/revdotcom/fstalign/pull/55/files#diff-63af1040726873c90ede631a5381b72f24d13087ac1ecb3587bf151675b4e871R555
The goal of moving the wer tag to a new struct is to allow the intended formatting for SBS, but still make it easy to output just the tag ID to the NLP output

**Review by qmac — APPROVED** [review]

**Pull request merged**